packages/story-editor/src/components/panels/panel/shared/title.js

```diff
@@ -229,6 +252,11 @@ function Title({
         >
           {children}
         </Heading>
+        {count !== undefined && (
```

Comment:
Since you do `count={layers?.length}` it means `count` can also be `null`. And we don't really want to accidentally render `null` here.

I'd do

```suggestion
        {Boolean(count) && (
```

instead

Reply:
`0` is a possibility here too, no? I guess we always have a background layer so maybe we don't have to worry about the `0` case

This is also on the base `panel` so trying to make it agnostic to the layer panel.

Maybe this to account for the `0` case:

```js
(count === 0 || Boolean(count)) && ...
```